% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/default_params_doc.R
\name{default_params_doc}
\alias{default_params_doc}
\title{This function does nothing. It is intended to inherit is parameters'
documentation.}
\usage{
default_params_doc(
  alignment,
  alignment_params,
  bd_mutation_rate,
  bd_tree,
  bd_tree_filename,
  brts,
  chain_length,
  clock_model,
  cond,
  crown_age,
  error_measure_params,
  experiment_type,
  experiments,
  fasta_filename,
  filename,
  folder_name,
  init_speciation_rate,
  init_extinction_rate,
  lambda,
  lambdas,
  mbd_l_matrix,
  mbd_mutation_rate,
  mbd_params,
  mbd_paramses,
  mbd_params_interval,
  mbd_sim_rng_seed,
  mbd_tree,
  mcmc_chain_length,
  misc_params,
  model_type,
  mu,
  mus,
  mutation_rate,
  n_replicates,
  nu,
  nus,
  nu_events,
  parameters,
  parameter_filename,
  parameters_filename,
  phylo,
  pir_params,
  posterior_trees,
  precision,
  project_folder_name,
  q,
  qs,
  razzo_params,
  razzo_paramses,
  rng_seed,
  sample_interval,
  seed,
  seed_folder,
  sequence_length,
  sim_pars,
  sim_phylo,
  site_model,
  sub_chain_length,
  tree,
  tree_type,
  tree_filename,
  trees_filename,
  twinning_params,
  verbose
)
}
\arguments{
\item{alignment}{a DNA alignment}

\item{alignment_params}{parameters for creating an alignment,
as can be created by \code{\link[pirouette]{create_alignment_params}}}

\item{bd_mutation_rate}{the mutation rate when creating an alignment
from a BD tree}

\item{bd_tree}{a phylogent of class \code{phylo}, created by a Birth Death
process}

\item{bd_tree_filename}{name of the file that stores a BD twin tree}

\item{brts}{set of branching times}

\item{chain_length}{something}

\item{clock_model}{Name of the clock model that has
to be used for the inference. Valid names are 'strict' and 'rln'.}

\item{cond}{conditioning as specified in \code{\link[mbd]{mbd_sim}}}

\item{crown_age}{The crown age of the tree.}

\item{error_measure_params}{parameters to set how the error
between given tree and inferred trees in measure,
as can be created by \code{\link[pirouette]{create_error_measure_params}}}

\item{experiment_type}{the type of experiment,
can be either \code{test} or \code{full}}

\item{experiments}{provides information on how the experiment has to be set
up. To create it use \code{\link[pirouette]{create_experiment}}}

\item{fasta_filename}{name of a FASTA file}

\item{filename}{the file's name, without the path}

\item{folder_name}{name of the main folder}

\item{init_speciation_rate}{a speciation rate}

\item{init_extinction_rate}{an extinction rate}

\item{lambda}{per-lineage speciation rate. See \code{\link[mbd]{mbd_sim}}}

\item{lambdas}{per-lineage speciation rates. See \code{\link[mbd]{mbd_sim}}}

\item{mbd_l_matrix}{the L matrix of an MBD tree}

\item{mbd_mutation_rate}{the mutation rate when creating an alignment
from a MBD tree}

\item{mbd_params}{MBD parameter set}

\item{mbd_paramses}{data frame with one MBD parameter set per row.
If a simulation with one MBD parameter set has 10 replicates,
this data frame will hold 10 rows, each with different seeds}

\item{mbd_params_interval}{data frame with one MBD parameter set per row.
If a simulation with one MBD parameter set has 10 replicates,
this data frame will hold 10 rows, each with different seeds}

\item{mbd_sim_rng_seed}{rng seed to simulate a mbd tree}

\item{mbd_tree}{an MBD tree}

\item{mcmc_chain_length}{length of an MCMC}

\item{misc_params}{additional parameters for razzo. They contain
tree_filename to store the original given tree and mbd_sim_rng_seed for
when an mbd tree is simulated}

\item{model_type}{the type of inference model, must be
\code{generative} or \code{candidate}}

\item{mu}{per-species extinction rate. See \code{\link[mbd]{mbd_sim}}}

\item{mus}{per-species extinction rates. See \code{\link[mbd]{mbd_sim}}}

\item{mutation_rate}{something}

\item{n_replicates}{number of replicates}

\item{nu}{the rate at which a multiple-birth specation is triggered.
See \code{\link[mbd]{mbd_sim}}}

\item{nus}{the rates at which a multiple-birth specation is triggered.
See \code{\link[mbd]{mbd_sim}}}

\item{nu_events}{the number of nu-triggered events that have to be
present in the simulated tree}

\item{parameters}{the razzo parameters}

\item{parameter_filename}{full path to a \code{parameters.RDa} file}

\item{parameters_filename}{full path to a \code{parameters.RDa} file}

\item{phylo}{a phylogeny}

\item{pir_params}{a parameter set for one \code{pirouette} run,
as created by \link[pirouette]{create_pir_params}.
Use \link{create_test_razzo_pir_params} to create
a \code{pir_params} that has the file naming scheme as \code{razzo}}

\item{posterior_trees}{phylogenetic trees in a BEAST2 posterior,
of class \code{multiphylo}}

\item{precision}{define the precision of the approximation.}

\item{project_folder_name}{project folder name,
will be the full path to \code{razzo_project}}

\item{q}{per-species speciation probability in case of occurrance of
a multiple event. See \code{\link[mbd]{mbd_sim}}}

\item{qs}{per-species speciation probabilities in case of occurrance of
a multiple event. See \code{\link[mbd]{mbd_sim}}}

\item{razzo_params}{a parameter set for one \code{razzo} experiment,
as created by \link{create_params_razzo}
and used by \link{run_razzo}}

\item{razzo_paramses}{a list of \code{razzo_params} for a
set of \code{razzo} experiment.
Use create \link{create_razzo_paramses} or
\link{create_test_razzo_paramses} to create such a list}

\item{rng_seed}{a random number generator seed}

\item{sample_interval}{the interval at which the MCMC algorithm
makes a measurement}

\item{seed}{a random number generator seed}

\item{seed_folder}{the folder where the data for a specific seed is stored}

\item{sequence_length}{the length of each DNA sequence in an alignment}

\item{sim_pars}{something}

\item{sim_phylo}{something}

\item{site_model}{Name of the site model that
has to be used for the inference. Valid names are 'jc69' and 'gtr'.}

\item{sub_chain_length}{length of the sub-chain used by the Nested Sampling
algorithm to estimate the marginal likelihood}

\item{tree}{an ultrametric phylogenetic tree of class \code{phylo}}

\item{tree_type}{can be 'true' or 'twin'}

\item{tree_filename}{name of the phylogeny file}

\item{trees_filename}{name of the BEAST2 posterior phylogenies file}

\item{twinning_params}{parameters for creating a twin tree,
as can be created by \code{\link[pirouette]{create_twinning_params}}}

\item{verbose}{give more output}
}
\description{
This function does nothing. It is intended to inherit is parameters'
documentation.
}
\note{
This is an internal function, so it should be marked with
  \code{@noRd}. This is not done, as this will disallow all
  functions to find the documentation parameters
}
\author{
Documentation by Giovanni Laudanno,
  use of this function by Richel J.C. Bilderbeek
}
